spec/models/request/sample_compound_aliquot_transfer_spec.rb

```diff
@@ -76,7 +78,20 @@
       expect(sequencing_request.target_asset.aliquots.first.library_type).to eq('Standard')
       expect(sequencing_request.target_asset.aliquots.first.study).to eq(study)
       expect(sequencing_request.target_asset.aliquots.first.project).to eq(project)
+      expect(sequencing_request.target_asset.aliquots.first.library_id).to eq(54)
       expect(sequencing_request.target_asset.samples.first.component_samples.order(:id)).to eq(samples.sort)
     end
+
+    # How the library_id should be set if the source aliquots have different library_ids is not defined
+    # Therefore, set it to null for now, until we have a real requirement
```

Comment:
I had a question about exactly this when reading the method, which this comment answers 👍 in that case maybe add this comment to the `_copy_library_id` method too? And could update the comment to `nil` not `null`